.travis.yml

```diff
@@ -71,7 +71,10 @@ matrix:
 
          - python: 3.6
            stage: Comprehensive tests
-           env: SETUP_CMD='test --online --coverage -V'
+           # Run this with pytest because our `setup.py` runner does not
+           # currently pick up doctests in docs/
+           env: MAIN_CMD="py.test"
+                SETUP_CMD="--remote-data -k-figure --cov ./"
```

Comment:
what -k-figure

Reply:
excludes tests with `pytest.mark.figure`

Reply:
and why do you skip those from the coverage?

Reply:
hmmm that's a fair point. The main reason is because we have hard pinned envs for the figure tests as we just compare hashes not images. I will give this some more thought in future test trickery.